build/unix/libc.modulemap

```diff
@@ -41,3 +41,9 @@ module "xlocale.h" [system] {
   export *
   header "xlocale.h"
 }
+
+// System header that we have difficult with merging.
+module "sys_types.h" [system] {
```

Comment:
Shouldn't we make an umbrella for `sys` and move it in the main module?

Reply:
We can properly fix it with the modulemap mounting script later. Right now placing an umbrella here breaks everything as the headers in `sys/` are partly 32-bit only:

`/usr/include/sys/vm86.h:24:3: error: This header is unsupported on x86-64.`

Let alone all the systems that have maybe other broken headers in there for some reason that an umbrella would pick up.

Reply:
Ah, ok.